packages/usa-tooltip/src/test/tooltips.spec.js

```diff
@@ -53,6 +80,22 @@ tests.forEach(({ name, selector: containerSelector }) => {
       assert.strictEqual(tooltipBody.classList.contains("is-set"), false);
     });
 
+    it("tooltip is visible on mouseover", () => {
+      EVENTS.mouseover(tooltipTrigger);
+      assert.strictEqual(tooltipBody.classList.contains("is-set"), true);
+    });
+
+    it("tooltip is hidden on mouseout", () => {
+      EVENTS.mouseout(tooltipTrigger);
+      assert.strictEqual(tooltipBody.classList.contains("is-set"), false);
+    });
```

Comment:
> [!note]
> Added some missing tests while I was in there

Reply:
Update: I moved these tests to #5919 instead since they felt more appropriate there.